azureDatabaseUtils/src/main/java/bio/terra/workspace/azureDatabaseUtils/database/DatabaseService.java

```diff
@@ -5,16 +5,15 @@
 import bio.terra.workspace.azureDatabaseUtils.storage.BlobStorage;
 import bio.terra.workspace.azureDatabaseUtils.validation.Validator;
 import com.azure.identity.extensions.jdbc.postgresql.AzurePostgresqlAuthenticationPlugin;
-import java.io.IOException;
-import java.io.InputStream;
+
+import java.io.*;
```

Comment:
FYI, I was told by someone on the workspaces team that they're trying to avoid importing with `.*`. Sounds like it's just a team style preference